README.md

```diff
@@ -1,91 +1,66 @@
+* **Produce PP and ADPP** dataset formats of the resulting parameters
+* **Save your analysis settings** and reupload them later to keep on analysing!
 
+## Installation
+To run the application, clone the repository and load it locally using the following commands:
+``` sh
+git clone https://github.com/pharmaverse/aNCA.git # Clone the repository
+cd aNCA # Navigate to the cloned directory
 ```
-
-### 3. Install the package
-
-
-Next make sure you have a package development environment set up. If not, install the `devtools` or a similar package (if it does not work, please use R version 4.3.3):
-
-```r
-options(repos = c(CRAN = "https://cloud.r-project.org/"))
-install.packages("devtools")
+You can then run the application from the R console:
+``` r
+# install devtools if not present
+if (!requireNamespace('devtools', quietly = TRUE)) install.packages('devtools')
+devtools::load_all() # load all dependencies
+aNCA::run_app() # run the applicaiton
 ```
 
+## Quick start
+The testing data will be automatically loaded upon application startup. You can provide your own dataset in the **data** tab. Here you can also specify pre-processing filters.
 
-If not already done, head into the cloned repository and install the package, e.g.:
+In the **NCA** tab, start off by loading the pre-processed data using *Submit* button. You will also need to choose dose number in the *Settings*. Then, you will be able to run the NCA analysis. From there, you can also specify different analysis options, like applying flag rule sets and selecting slopes.
 
+After the setup is done and analysis is performed, you are free to explore the results in the **Outputs** tab. Application supports various customizable plots, as well as report exporting.
 
-```r
-devtools::install()
-```
+## Contributing 
+### As developer 
+To ensure a clean codebase and smooth cooperation, please adhere to the [contributing guidelines](CONTRIBUTING.md).
 
+### As user 
+Feel free to open identified issues, to reach out to us for questions or fill our [google sheet]() for feedback.
 
-## Use
+## Documentation and references
 
-Once correctly installed, the aNCA app can be simply run with the following two lines of code: 
+* Please go to [our Website](https://pharmaverse.github.io/aNCA/) for further information on the **aNCA app** (still in development).
+* The main package used by the App is `PKNCA`. You can find more of it on its [GitHub](https://github.com/billdenney/pknca)
 
 
-```r
-library(aNCA)
-devtools::load_all()
-aNCA::run_app()
-```
-
-Have fun running your tailored Non-Compartmental Analysis!
-
-<!--
-## Contribute as developer
-
-To ensure a clean and informative git version history, please adhere to the [guidelines](man/GUIDELINES.md) of our git workflow. You can find further information on possible ways to use gits full power on our homemade [cheatsheet](man/GIT-CHEATSHEET.md).
-
--->
-
 <!-- 
 
 ## Getting started
```

Comment:
issue: there is a lot of commented out content, please include whatever is relevant and remove the rest.